### ISSUE TITLE
 Could not pass event StructureGrowEvent to WorldGuardExtraFlags v4.1.5-SNAPSHOT

### ISSUE THREAD

**Issue by l3unnyAnoF4** [Invvk/WorldGuardExtraFlagsPlugin#1]
```
[00:33:51 ERROR]: Could not pass event StructureGrowEvent to WorldGuardExtraFlags v4.1.5-SNAPSHOT
java.lang.NullPointerException: null
```

https://streamable.com/anjyyw

**Comment by Invvk** [comment]
Hello, First sorry for the late response I've been busy.
As for this issue I'm fully aware of it. However, I'm currently re-coding the plugin and it could take a while for a release.

Best Regards,
Invvk.

**Comment by Invvk** [comment]
the issue has been fixed in ec0a9f4
@l3unnyAnoF4

**Comment by l3unnyAnoF4** [comment]
So how do I use RESPAWN-LOCATION?
